src/components/molecules/Navbar/index.jsx

```diff
@@ -1,3 +1,4 @@
+//#region > Imports
 //> React
 // Contains all the functionality necessary to define React components
 import React from "react";
```

Comment:
Remove the newline below this line. (cannot be marked)

Reply:
We split imports per header and a new line.